CSUN-Comp490/RealTimeCaptioning
# Rtc 85

In this pull request there were a couple things that got added.

- Captionists can now create new transcripts that correspond to a given class and title it however they want
- Captionists can now edit a transcript after it has been saved.
- Captionists can delete transcripts 
- A new route was created in the backend to get a class by courseID
- A new route was created for viewing the transcripts

To test:
navigate to the backend directory and run `seed`
run the application with `ng serve` and `nodemon`
To create a new transcript go to localhost:4200/captioner-session/12345 (or any other id)
Type away and it should be saved to the dashboard.
